bookwyrm/settings.py

```diff
@@ -233,3 +233,7 @@
     MEDIA_FULL_URL = f"{PROTOCOL}://{DOMAIN}{MEDIA_URL}"
     STATIC_FULL_URL = f"{PROTOCOL}://{DOMAIN}{STATIC_URL}"
     MEDIA_ROOT = os.path.join(BASE_DIR, env("MEDIA_ROOT", "images"))
+
+OTEL_EXPORTER_OTLP_ENDPOINT = env("OTEL_EXPORTER_OTLP_ENDPOINT")
```

Comment:
Can you add default values (presumably of `None`) to these? That way there won't be an error in `bookwyrm/apps.py` for installations where the variables are absent from the env file -- otherwise admins will have to manually copy over the lines from `.env.prod.example` to .`env`